python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_decorator.py

```diff
@@ -1060,6 +1060,27 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
     assert result.success
 
 
+@pytest.mark.skipif(
+    version.parse(dbt_version) < version.parse("1.8.0"),
+    reason="dbt unit test support is only available in `dbt-core>=1.8.0`",
+)
+def test_dbt_with_unit_tests_with_tags_selector(
```

Comment:
can we just parametrize the test above by `select`?

Then we have two options: `select: ["fqn:*", "tag:test"]`

Reply:
Done in [5d3fee4](https://github.com/dagster-io/dagster/pull/23232/commits/5d3fee41fb2333d37016a8b5f7cd565d8c875774)